app/_layouts/collection.njk

```diff
@@ -33,33 +34,46 @@
         prose: content
       }) }}
       <div class="govuk-grid-row">
-        <section class="govuk-grid-column-two-thirds">
-          <h2 class="govuk-heading-l govuk-!-font-size-27">Posts</h2>
-          {{ appDocumentList({
-            classes: "govuk-!-margin-bottom-6",
-            items: pagination.items
-          }) }}
-          {{ appPagination({
-            previous: {
-              text: "Previous",
-              href: pagination.href.previous
-            } if pagination.href.previous,
-            next: {
-              text: 'Next',
-              href: pagination.href.next
-            } if pagination.href.next,
-            selected: pagination.pageNumber,
-            items: pagination.hrefs
-          }) }}
-        </section>
-
-        {% if related %}
-        <div class="govuk-grid-column-one-third">
-          {{ appRelated({
-            title: related.title or "Related links",
-            items: related.items
-          }) }}
-        </div>
+        {% if pagination.items %}
```

Comment:
I wonder if this forking logic needs an explanatory comment.

eg Show a list of items if using a collection with pagination, otherwise show sections, or something.

Reply:
Or use of a temporary variable, `{% if listPosts %}{% elif listServices %}{% endif %}`